# Generated by Django 3.2.4 on 2021-07-14 04:51

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('Exceptions', '0002_auto_20210713_1734'),
    ]

    operations = [
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_BusinessLine',
            field=models.CharField(default='', max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_COBDT',
            field=models.DateField(default='2000-01-01'),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_LegalEntity',
            field=models.CharField(default='', max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_ProfitCenter',
            field=models.CharField(default='', max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_Region',
            field=models.CharField(default='', max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_component',
            field=models.CharField(default='', max_length=20, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_description',
            field=models.CharField(default='', max_length=50, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_level',
            field=models.CharField(default='', max_length=20, null=True),
        ),
        migrations.AlterField(
            model_name='exceptiontype',
            name='exception_name',
            field=models.CharField(default='', max_length=50, null=True),
        ),
    ]
